index.html

```diff
@@ -40,7 +40,7 @@
     <ul class="sidebar-nav">
         <a id="menu-close" href="#" class="btn btn-light btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
         <li class="sidebar-brand">
-            <a href="#top">MalagaMakers</a>
+            <a href="#top">Malaga<wbr>Makers</a>

```

Comment:
What `<wbr>` really means here is word break *opportunity*. It means that, only in extreme cases  (small screens of e-watches, for example) we prefer to show:
`Malaga\n
Makers`
rather than `MalagaMak\n
ers`. Or something worst. 
 😊